docs/ref/conf.rst

```diff
@@ -193,3 +193,21 @@ localeDir
 Default: ``<rootDir>/locale``
 
 Directory where message catalogs should be saved.
+
+
+sorting
```

Comment:
Considering if `orderBy` is a better name for this

Reply:
I like `orderBy` 👍